src/components/controls/dates/Calendar/Calendar.tsx

```diff
@@ -18,6 +18,9 @@ interface CalendarBaseProps {
 
   /** Props passed to root element. */
   wrapperProps?: BoxProps
+
+  /** Initial date (month and year) to be displayed. */
```

Comment:
What do you think about this?

```suggestion
  /** Initial month displayed. */
```

Reply:
I think that makes sense because in this context you are not referring to month as in "January", but a specific month in time as in "January 2023". It's kind of the same for example in the calendar where you can switch between months, which also include a year implicitly. Makes sense?
<img width="321" alt="image" src="https://user-images.githubusercontent.com/44197016/222139036-44909d0b-a8c5-4fa5-ac2c-aec9d2330e40.png">